### PR TITLE
add format option to dashboards

### PR DESCRIPTION
**Pull request by bspindler**

### CODE REVIEW

File: ts/commands/DashboardCommands.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ class DashboardCommands {
       .option('--username', 'Metricly Username')
       .option('--password', 'Metricly Password')
       .option('--profile', 'Metricly profile', /.*/, 'default')
+      .option('--format', 'output format: text, json', /.*/, 'text')
```
Comment (TheConnMan) [inline ts/commands/DashboardCommands.ts]: You can define available strings in an array using a built in array validator: https://www.npmjs.com/package/caporal#array-validator

Comment (bspindler) [reply ts/commands/DashboardCommands.ts]: do you know how to define the default with that approach?

Comment (TheConnMan) [reply ts/commands/DashboardCommands.ts]: Not sure, though I'd either change the if statements into if/else when using it, or assign a default if the option is null right inside the initializer a couple logs down.

Comment (bspindler) [reply ts/commands/DashboardCommands.ts]: done

File: ts/commands/ConfigCommands.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ class ConfigCommands {
           name: 'password',
           type: 'password'
         }, {
-          default: profileConfig.endpoint || 'https://app.netuitive.com',
+          default: profileConfig.endpoint || 'https://app.metricly.com',
```
Comment (TheConnMan) [inline ts/commands/ConfigCommands.ts]: This change is unrelated to this PR, right?

Comment (bspindler) [reply ts/commands/ConfigCommands.ts]: yes aesthetics only

Comment (TheConnMan) [reply ts/commands/ConfigCommands.ts]: Ok, you should just push this type of change directly to `develop` instead of lumping it into an unrelated commit and PR.
